### PR TITLE
[Doc Improvement][Types of cards][3437956]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/73900b21a1466ef98d20949ed76d8ad286a2e3d9/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52fbf16e-960f-4fa6-8d46-bc4ad070b845) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 1]
```diff
@@ -126,8 +126,9 @@ The following table provides the features that support Adaptive Cards:
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for bot sent cards and action based message extensions.
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for other capabilities, such as cards sent by user (search based message extensions and link unfurling), tabs, and task modules.
 > * Positive or destructive action styling is not supported in Adaptive Cards on the Teams platform.
-> * Media elements are currently not supported in Adaptive Card on the Teams platform.
+> * Media elements are not supported in Adaptive Card on the Teams platform.
 > * Test your full width Adaptive Card in narrow form factors such as mobile and meeting side panels to ensure that content is not truncated.
+> * Adaptive Cards in Teams don't support file or image upload capability.
```
Comment (v-deabira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:131]: It's not a capability.

**Review by v-deabira — CHANGES_REQUESTED** [review]

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/task-modules-and-cards/cards/cards-reference.md:131]: Updated as Adaptive Cards in Teams don't support file or image upload.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b3918f3628c425fb5f201171e155d33be901c2bd/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/43132f9a-34bd-4b2b-9bb6-225df2a22b1f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 2]
```diff
@@ -126,8 +126,9 @@ The following table provides the features that support Adaptive Cards:
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for bot sent cards and action based message extensions.
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for other capabilities, such as cards sent by user (search based message extensions and link unfurling), tabs, and task modules.
 > * Positive or destructive action styling is not supported in Adaptive Cards on the Teams platform.
-> * Media elements are currently not supported in Adaptive Card on the Teams platform.
+> * Media elements are not supported in Adaptive Card on the Teams platform.
```
Comment (v-deabira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:129]: On the Teams platform, Adaptive Card does not support media elements.

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md [hunk 3]
```diff
@@ -126,8 +126,9 @@ The following table provides the features that support Adaptive Cards:
...
 > * Test your full width Adaptive Card in narrow form factors such as mobile and meeting side panels to ensure that content is not truncated.
+> * Adaptive Cards in Teams don't support file or image upload.
```
Comment (v-deabira) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:131]: Adaptive Cards within Teams don't provide support for file or image uploads.

**Review by v-deabira — CHANGES_REQUESTED** [review]

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 2 above)
Comment (Wajeed-msft) [inline msteams-platform/task-modules-and-cards/cards/cards-reference.md:129]: We should remove this line. 

https://learn.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/media-elements-in-adaptive-cards?tabs=desktop

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 2 above)
Comment (v-bvishnu) [reply msteams-platform/task-modules-and-cards/cards/cards-reference.md:129]: Updated.

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 3 above)
Comment (v-bvishnu) [reply msteams-platform/task-modules-and-cards/cards/cards-reference.md:131]: Updated.

File: msteams-platform/task-modules-and-cards/cards/cards-reference.md (hunk 2 above)
Comment (v-bvishnu) [reply msteams-platform/task-modules-and-cards/cards/cards-reference.md:129]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c1bb31f273617516f27305c2bf66d8a489813101/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d7c77993-7a6c-4aa5-89ba-eae33664395e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4cd2713a3573303c388ebb8e3770c0d5cdee118f/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62725306-53f3-4bed-8cf7-acd6cd814e69) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/de9e860ee57ae46eeb9f20d2136bbf749111f6f7/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0322b03a-bbd3-44e6-87bd-f606ec1d6488) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dcd00386cdaa8cec0569729a91645b2b321e7030/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-reference.md) | 93 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2e43bf28-b251-4a96-8b24-8f17345b9ef2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**